src/amo/components/AddonsByAuthorsCard/index.js

```diff
@@ -359,25 +373,29 @@ export class AddonsByAuthorsCardBase extends React.Component<InternalProps> {
 export const mapStateToProps = (state: AppState, ownProps: Props) => {
   const { addonType, authorIds, forAddonSlug, numberOfAddons } = ownProps;
 
-  let addons = getAddonsForAuthorIds(
-    state.addonsByAuthors,
-    authorIds,
-    addonType,
-    forAddonSlug,
-  );
-  addons = addons ? addons.slice(0, numberOfAddons) : addons;
+  let addons = [];
+  let loading = true;
```

Comment:
When you set `loading` explicitly to `true` here, it is out of sync with the Redux state which seems wrong. Maybe the Redux state needs fixing instead? Or were you trying to show loading bars before it begins loading? If so, why?

Reply:
This component is used for secondary data only. I want it to have a "loading state" upon construction, so that it can always be rendered.

Currently, we need the `user` to render it, which means a condition to render/not render it. I want to always render it.